.Rbuildignore

```diff
@@ -8,3 +8,5 @@
 ^\.travis\.yml$
 ^cran-comments\.md$
 ^codecov\.yml$
+.V8history
```

Comment:
move `.V8history` into `.gitignore`

Reply:
Done

Reply:
√. 

Feel free to hit `resolve` on GitHub.  (Then all the comments disappear)